packages/tooltips/src/elements/Tooltip.spec.tsx

```diff
@@ -12,6 +12,7 @@ import { PALETTE } from '@zendeskgarden/react-theming';
 import { Tooltip } from './Tooltip';
 import { ITooltipProps } from '../types';
 
+// TODO: remove once `getRenderFn` is in `garden-test-utils`
```

Comment:
To do once https://github.com/zendeskgarden/react-components/pull/1819 has merged.